TypeCobol.Test/Parser/Programs/Cobol85/EmptyArray.Mix.txt

```diff
@@ -0,0 +1,292 @@
+      *   "PICTURE X(1)"was assumed.
+Line 52[15,26] <27, Error, Syntax> - Syntax error : A "PICTURE"clause was not found for elementary item "Array-A09-KO"."PICTURE X(1)"was assumed.
+Line 52[15,26] <27, Error, Syntax> - Syntax error : A group item cannot be empty.
+           05 Array-A09-KO OCCURS 50 packed-decimal.
+      *The"PICTURE"clause for item"Array-A10-KO"was not compatible with the
+      *   specified"USAGE"."USAGE DISPLAY"was assumed.
+      *A"PICTURE"clause was not found for elementary item"Array-A10-KO".
+      *   "PICTURE X(1)"was assumed.
+Line 57[15,26] <27, Error, Syntax> - Syntax error : A "PICTURE"clause was not found for elementary item "Array-A10-KO"."PICTURE X(1)"was assumed.
+Line 57[15,26] <27, Error, Syntax> - Syntax error : A group item cannot be empty.
+           05 Array-A10-KO OCCURS 50 comp.
+           05 Array-A11-OK OCCURS 50 comp-1.
+           05 Array-A12-OK OCCURS 50 comp-2.
+
+      *The"PICTURE"clause for item"Array-A13-KO"was not compatible with the
+      *   specified"USAGE"."USAGE DISPLAY"was assumed.
+      *A"PICTURE"clause was not found for elementary item"Array-A13-KO".
+      *   "PICTURE X(1)"was assumed.
+Line 65[15,26] <27, Error, Syntax> - Syntax error : A "PICTURE"clause was not found for elementary item "Array-A13-KO"."PICTURE X(1)"was assumed.
+Line 65[15,26] <27, Error, Syntax> - Syntax error : A group item cannot be empty.
+           05 Array-A13-KO OCCURS 50 comp-3.
+      *The"PICTURE"clause for item"Array-A14-KO"was not compatible with the
+      *   specified"USAGE"."USAGE DISPLAY"was assumed.
+      *A"PICTURE"clause was not found for elementary item"Array-A14-KO".
+      *   "PICTURE X(1)"was assumed.
+Line 70[15,26] <27, Error, Syntax> - Syntax error : A "PICTURE"clause was not found for elementary item "Array-A14-KO"."PICTURE X(1)"was assumed.
+Line 70[15,26] <27, Error, Syntax> - Syntax error : A group item cannot be empty.
+           05 Array-A14-KO OCCURS 50 comp-4.
+      *The"PICTURE"clause for item"Array-A15-KO"was not compatible with the
+      *   specified"USAGE"."USAGE DISPLAY"was assumed.
+      *A"PICTURE"clause was not found for elementary item"Array-A15-KO".
+      *   "PICTURE X(1)"was assumed.
+Line 75[15,26] <27, Error, Syntax> - Syntax error : A "PICTURE"clause was not found for elementary item "Array-A15-KO"."PICTURE X(1)"was assumed.
+Line 75[15,26] <27, Error, Syntax> - Syntax error : A group item cannot be empty.
+           05 Array-A15-KO OCCURS 50 comp-5.
+           05 Array-A16-OK OCCURS 50 pointer.
+           05 Array-A17-OK OCCURS 50 pointer-32.
+           05 Array-A18-OK OCCURS 50 procedure-pointer.
+           05 Array-A19-OK OCCURS 50 function-pointer.
+
+      *A"PICTURE"clause was not found for elementary item"ARRAY-A20-KO".
+      *   "PICTURE X(1)"was assumed.
+           05 Counter pic 9.
+Line 84[15,26] <27, Error, Syntax> - Syntax error : A "PICTURE"clause was not found for elementary item "Array-A20-KO"."PICTURE X(1)"was assumed.
```

Comment:
Can you check why we don't have the error "A group item cannot be empty." in this case ?

Reply:
This is because "**Array-A20-KO**" is having a child which is '**MyIdxCustom**'.
So test in line 820 is false and the error is not added.

![image](https://github.com/TypeCobolTeam/TypeCobol/assets/129841296/fea43172-e26a-47da-a731-21a5ee7b089b)

Reply:
An error is now raised when an OCCURS has only INDEX children